### DISCUSSION TITLE
DeprecationWarning: `general_plain_validator_function` is deprecated

### DISCUSSION THREAD

**Discussion by tnguyen-LiveData** [fastapi/fastapi#10339] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import fastapi
```

### Description

* Run `python -W error`
* `import fastapi`
* Getting the following error: 
 ```
DeprecationWarning: `general_plain_validator_function` is deprecated, use `with_info_plain_validator_function instead.
```

### Operating System

Linux, Windows

### FastAPI Version

0.103.1

### Pydantic Version

2.4.1

### Python Version

Python 3.11.5

**Comment by priyanshu-panwar** [comment]
Hey
I could not reproduce your issue.
```
Python 3.11.5
fastapi              0.103.1
pydantic             2.4.1
```
```
C:\Users\Terminal>python -W error
Python 3.11.5 (tags/v3.11.5:cce6ba9, Aug 24 2023, 14:38:34) [MSC v.1936 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import fastapi
>>> exit()
```
No warnings

**Reply by Ryandaydev** [reply, marked as answer]
I can't reproduce this issue, so I can't test this.

However, I am pretty sure those errors are related to updates to Pydantic. They may have been resolved by the latest release 0.103.2 according to the release notes here:
https://fastapi.tiangolo.com/release-notes/ 

So you may want to re-install fastapi 0.103.2.

pip install fastapi==0.103.2

**Reply by tnguyen-LiveData** [reply]
You're correct @Ryandaydev 
The issue was addressed in PR `https://github.com/tiangolo/fastapi/pull/10344`
